.github/workflows/ci.yml

```diff
@@ -82,23 +82,6 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Save PR number
-        env:
-          PR_NUMBER: ${{ github.event.number }}
-          PR_SHA: ${{ github.event.pull_request.head.sha }}
-          MERGE_SHA: ${{ github.sha }}
-        run: |
-          mkdir -p ./pr
-          echo $PR_NUMBER > ./pr/pr_number
-          echo $PR_SHA > ./pr/pr_sha
-          echo $MERGE_SHA > ./pr/merge_sha
-
-      - name: Upload Artifact
-        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8
-        with:
-          name: pr_number
-          path: pr/
-
```

Comment:
This feels unrelated to the other change.  Am I missing something?  If not, can this be split into two separate PRs so they can be discussed independently?

Reply:
Oh, I see.  Previously we saved an artifact and then used that to determine whether to comment or not, but now we comment in the same workflow so it is easier to see results and comment accordingly.

Reply:
This also means that the rate limit is lower.